src/index.js

```diff
@@ -0,0 +1,30 @@
+require('./index.css')
+const inject = require('./inject')
+const changeColor = require('./change-color')
+
+// invoke inject function
+inject()
+
+/**
+ * look for changes in the js-pjax-loader-bar
+ * and invoke inject function
+ * to inject github streak container
+ */
+const observer = new MutationObserver((mutations) => {
+    mutations.forEach((mutation) => {
+        if (mutation.oldValue && mutation.oldValue.indexOf("is-loading") > -1 ) {
+            // invoke inject function
+            inject()
+        }
+    })
+})
```

Comment:
mutation observer logic is now simple, it aways observes `js-pjax-loader-bar` and invokes `inject` if desired.